setup-kubehound-test-cluster.sh

```
@@ -0,0 +1,171 @@
+
+# Source shared logging functions
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+source "$SCRIPT_DIR/scripts/common.sh"
+
+# Configuration
+CLUSTER_NAME="kubehound.test.local"
+KUBEHOUND_REPO="/tmp/kubehound-repo"
+KUBECONFIG_FILE="./kubehound-test.kubeconfig"
+
+main() {
+    local start_time=$(date +%s)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)